build.sbt

```diff
@@ -255,6 +255,8 @@ lazy val akkaVer = Def.setting[String] {
   }
 }
 
+lazy val pekkoVer = Def.setting[String]("1.0.1")
+
 val akkaContribVer = Def.setting[String] {
```

Comment:
`akka-stream-contrib` is actually not used if I am not mistaken? Thats why I skipped it in Pekko.

Reply:
```scala
final class AkkaStreamSpec(implicit ee: ExecutionEnv) extends org.specs2.mutable.Specification {

  "Akka Stream".title

  implicit lazy val system: ActorSystem = ActorSystem("anorm-tests")

  implicit def materializer: Materializer =
    akka.stream.ActorMaterializer.create(system)

  // Akka-Contrib issue with Akka-Stream > 2.5.4
  // import akka.stream.contrib.TestKit.assertAllStagesStopped
  def assertAllStagesStopped[T](f: => T) = f

```

Reply:
Sorry I am confused. 😢 

Is it fine to skip akka-contrib? From what I understand it is not a binary but a library of source code. How is it used in Anorm code and why do we import it? 🤔

Reply:
Rather re-enable contrib in akka-stream and support alternative for pekko, to ensure there is no stream leak

Reply:
I did the changes, imported it from `akka-stream-testkit`/`pekko-stream-testkit`